packages/search/src/App.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Switch, Route, Router } from 'react-router-dom';
+import {
+  StylesProvider,
+  createGenerateClassName,
+} from '@material-ui/core/styles';
+
+const generateClassName = createGenerateClassName({
+  productionPrefix: 'ma',
```

Comment:
we talked yesterday about this one, should be something different from `ma` right?